### ISSUE TITLE
A question about pre-trained weights

### ISSUE THREAD

**Issue by indersingh17188** [Alibaba-MIIL/ASL#54]
Hello,

Very nice work. I am highly inspired by it and working on training from the scratch. I have a question regarding the default pre-trained weights which you have used to initialize the training. For example: the one which you have used locally were I guess: 'tresnet_m.pth'. Are they the one which are shared in the repo at the link https://github.com/Alibaba-MIIL/ASL/blob/main/MODEL_ZOO.md

When I am starting the training from scratch without any pre-trained weights, my loss is extremely high and the val mAP is never crossing 50 or so even after 80 epochs.

Thanks in advance for your time.

Best,
Inder

**Comment by mrT23** [comment]
pretraining is a key component in transfer learning.

when you initialize your model from scratch, the training script should be very different (for example - higher learning rates, longer training). even after these adjustments, you probably won't reproduce the scores obtain when using pertaining. 

so  - use pretraining for transfer learning. always.

regarding pretrain weights, for TResNet-L we did use in the article the weights from https://github.com/Alibaba-MIIL/ASL/blob/main/MODEL_ZOO.md (ImageNet-1K pretraining).

Notice that we now offer weights from ImageNet-21K pretraining, which further improved scores by 1-2%
https://github.com/Alibaba-MIIL/ImageNet21K

**Comment by indersingh17188** [comment]
Thank you so much for your quick response. I really appreciate it.
The weights available at the given link are full trained weights which can reproduce almost same results as published. But are there any weights which were pre-trained on ImageNet only?

Thanks in advance for your time.

**Comment by mrT23** [comment]
in
https://github.com/Alibaba-MIIL/TResNet/blob/master/MODEL_ZOO.md
are model pretrained on ImageNet-1K

in
https://github.com/Alibaba-MIIL/ImageNet21K/blob/main/MODEL_ZOO.md
are model pretrained on ImageNet-21K
